modules/base/rotation/fixedrotation.cpp

```diff
@@ -190,7 +190,46 @@ namespace {
         openspace::properties::Property::Visibility::AdvancedUser
     };
 
+    // This `Rotation` calculates the rotation in such a way that the attached scene graph
+    // node will always be relative to some other direction or pointing at another scene
+    // graph node.
+    //
+    // The first use-case of the `FixedRotation` needs exactly two of its three axes
+    // (`XAxis`, `YAxis`, `ZAxis`) specified with the last axis being unspecified. The
+    // axis that is missing will be calculated using a right-handed coordinate completion
+    // using the two provided axes. Each axis can be specified either by providing the
+    // `Identifier` of another scene graph node, or by providing a direction vector. For
+    // the general use-case of this `Rotation`, one of the provided axes usually is an
+    // `Identifier` and the other is a direction vector (see the examples below).
+    // If any of the axes is using the `Identifier` of another scene graph node, the
+    // `Attached` value must be specfied and should almost always be the identifier of the
```

Comment:
```suggestion
    // `Attached` value must be specified and should almost always be the identifier of the
```

So great that you added the documentation for this btw 🙏